Cargo.toml

```diff
@@ -42,7 +42,7 @@ url = { version = "2.5.3", default-features = false, features = ["std"] }
 cargo = { level = "warn", priority = -1 }
 nursery = { level = "warn", priority = -1 }
 pedantic = { level = "warn", priority = -1 }
-multiple_crate_versions = "allow"
+get_unwrap = "warn"
```

Comment:
For others that wondered why we don't enable the entire `restriction` clippy lint group:

> The clippy::restriction group contains lints that will restrict you from using certain parts of the Rust language. It is not recommended to enable the whole group, but rather cherry-pick lints that are useful for your code base and your use case.

https://doc.rust-lang.org/clippy/lints.html#restriction